pages/container/simple.page.tsx

```diff
@@ -102,6 +102,10 @@ export default function SimpleContainers() {
               Content area takes the available vertical space
             </Container>
           </div>
+          <Container
+            disableContentPaddings={true}
```

Comment:
I would rather leave it here because without disableContentPaddings the border looks correct. It is to make sure the fix still works even with disableContentPaddings.